### PR TITLE
BXMSDOC 5023 PLANNER: create first draft emp rostering on IDE, after updates from SMEs

### PR DESCRIPTION
**Pull request by emmurphy1**
Details in jira:
[BXMSDOC-5023](https://issues.jboss.org/browse/BXMSDOC-5023)

### CODE REVIEW

File: doc-content/enterprise-only/employee-rostering/optashift-ER-overview-source-con.adoc [hunk 1]
```diff
@@ -21,26 +21,17 @@ The modules contain a number of files, including Java classes. This document lis
...
+** `src/main/java/org.optaweb.employeerostering.domain.skill/Skill.java`: Defines a skill that an employee can have.
+** `src/main/java/org.optaweb.employeerostering.domain.spot/Spot.java`: Defines a spot where employees can be placed. For example,  a `Kitchen` can be a spot.
+** `src/main/java/org.optaweb.employeerostering.domain.contract/Contract.java`:  Defines a contract that sets limits on work time for an employee in various time periods.
+** `src/main/java/org.optaweb.employeerostering.domain.tenant/Tenant.java`: Defines a tenant. Each tenant represents an independent set of data. Changes in the data for one tenant do not affect any other tenants.
+** `*View.java` classes related to domain objects do define value sets that are calculated from other information; the client application can read these values through the REST API, but not write them.
+** `*Service.java` interfaces located in the service package define the REST API. Both the server and the client application separately define implementations of these interfaces.
```
Comment (sterobin) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-overview-source-con.adoc]: For these last two (and I assume the asterisk is a wildcard), I think we should structure them consistently with all the ones above, like "`*View.java`: Classes that are related to ... and that define value ...", and same for the next one. And I'd reword "do define" to just define, and say "... other information. The client ..." with no semicolon. Clearer, and more consistent with the next one anyway.

**Review by sterobin — COMMENTED** [review]
@emmurphy1 Done. Be sure to expand any "hidden" comments in the PR. Some comments you might disregard if the way these files are sourced is unusual or something. Your call.

File: doc-content/enterprise-only/employee-rostering/optashift-ER-building-IntelliJ-IDEA-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+.Procedure
+. Start IntelliJ IDEA.
+. From the IntelliJ IDEA main menu, select *File* > *Open*.
+. Select the root directory of the application source and click *OK*.
+. From the main menu, select *Run* > *Edit Configurations*.
+. In the window that appears, expand *Templates* and select *Maven*. The Maven sidebar appears.
+. In the Maven sidebar, select `employee-rostering-backend` from the  *Working Directory* menu.
+. In *Command Line*, enter `spring-boot:run`.
+. To start the back end, click *OK* .
+. In a terminal window, navigate to the `employee-rostering-frontend` directory.
```
[the remark this answers is not in the document]
Comment (emmurphy1) [reply doc-content/enterprise-only/employee-rostering/optashift-ER-building-IntelliJ-IDEA-proc.adoc]: Ah yeah. That's a hard habit for me to break. But I agree. I'll change to 'In a command terminal'

File: doc-content/enterprise-only/employee-rostering/optashift-ER-modifying-proc.adoc [hunk 3]
```diff
@@ -21,7 +21,7 @@ The following procedure outlines the general approach to modifying the employee
...
+. When all the data is available, modify the rules. For most modifications, you must add a new rule. The rules are located in the `src/main/resources/org/optaweb/employeerostering/service/solver/employeeRosteringScoreRules.drl` file of the `employee-rostering-backend` module.
 +
-Use the Drools language for the rules. For reference information about the Drools rule language, see https://docs.jboss.org/drools/release/{COMMUNITY_VERSION}.0.Final/drools-docs/html_single/#_droolslanguagereferencechapter[Rule Language Reference]. Classes defined in the `optaweb-employee-rostering-shared` and `optaweb-employee-rostering-server` modules are available to the {DECISION_ENGINE}.
+Use the Drools language for the rules. For reference information about the Drools rule language, see https://docs.jboss.org/drools/release/{COMMUNITY_VERSION}.0.Final/drools-docs/html_single/#_droolslanguagereferencechapter[Rule Language Reference]. Classes defined in the `optaweb-employee-rostering-backend` modules are available to the {DECISION_ENGINE}.
```
[the remark this answers is not in the document]
Comment (emmurphy1) [reply doc-content/enterprise-only/employee-rostering/optashift-ER-modifying-proc.adoc]: Yep, that works. Thanks.
